server/routes/products.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+
+//:id will be list id each product is attached with a list
+
+//create
+router.route('/:listId(\\d+)').post(protect, (req,res)=>{res.send()});
```

Comment:
any code written for all of these endpoints rather than just `res.send()`? what do these endpoints do atm?

Reply:
that was not part of the task. it was to just create the routes